dts/bindings/pinctrl/ite,it8xxx2-pinctrl.yaml

```diff
@@ -0,0 +1,97 @@
+# Copyright (c) 2022 ITE Corporation. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+    The ITE IT8XXX2 pin controller is a node responsible for controlling
+    pin function selection and pin properties. For example, you can
+    use this node to route UART1 RX and TX setting the alternate
+    function on the pin.
+
+    The node has the 'pinctrl' node label set in your SoC's devicetree,
+    so you can modify it like this:
+
+      &pinctrl {
+              /* your modifications go here */
+      };
+
+    All device pin configurations should be placed in child nodes of the
+    'pinctrl' node, as shown in this example:
+
+      /* You can put this in places like a board-pinctrl.dtsi file in
+       * your board directory, or a devicetree overlay in your application.
+       */
+
+      /* include pre-defined pins and functions for the SoC used by the board */
+      #include <dt-bindings/pinctrl/it8xxx2-pinctrl.h>
+
+      &pinctrl {
+        /* configuration for I2C0 default state */
+        pinctrl_i2c_clk0: i2c_clk0 {
+                pinmuxs = <&pinctrlb 3 IT8XXX2_PINMUX_FUNC_1>;
+                gpio-voltage-1p8;
+        };
+        pinctrl_i2c_data0: i2c_data0 {
+                pinmuxs = <&pinctrlb 4 IT8XXX2_PINMUX_FUNC_1>;
+                gpio-voltage-1p8;
+        };
+        /* configuration for UART0 default state */
+        pinctrl_uart1_rx: uart1_rx {
+                pinmuxs = <&pinctrlb 0 IT8XXX2_PINMUX_FUNC_3>;
+                bias-pull-up;
+        };
+        pinctrl_uart1_tx: uart1_tx {
+                pinmuxs = <&pinctrlb 1 IT8XXX2_PINMUX_FUNC_3>;
+        };
+      };
+
+    The 'pinctrl_uart1_rx' child node encodes the pin configurations
+    for a particular state of a device; in this case, the default
+    (that is, active) sate.
+
+    To link pin configurations with a device, use a pinctrl-N property for some
+    number N, like this example you could place in your board's DTS file:
+
+      #include "board-pinctrl.dtsi"
+
+      &uart0 {
+        pinctrl-0 = <&pinctrl_uart1_rx &pinctrl_uart1_tx>;
+        pinctrl-1 = <&pinctrl_uart1_rx_sleep &pinctrl_uart1_tx_sleep>;
+        pinctrl-2 = <&pinctrl_uart1_rx_priv &pinctrl_uart1_tx_priv>;
+        pinctrl-names = "default", "sleep", "priv_start";
+      };
+
+compatible: "ite,it8xxx2-pinctrl"
+
+include:
+    - name: base.yaml
+    - name: pincfg-node.yaml
+      child-binding:
+        property-allowlist:
+          - bias-high-impedance
+          - bias-pull-up
+          - bias-pull-down
+          - input-enable
+          - gpio-voltage-1p8
+          - gpio-voltage-3p3
```

Comment:
these are not standard (I can't see them in `pincfg-node.yaml`)

Reply:
Removed.